app/grandchallenge/components/models.py

```diff
@@ -1143,6 +1150,14 @@ def _clean_store_in_database(self):
                 f"Interface {self.kind} objects cannot be stored in the database"
             )
 
+    def _clean_example_value(self):
+
+        if self.example_value:
+            civ = ComponentInterfaceValue(
+                interface=self, value=self.example_value
+            )
+            civ.clean()
```

Comment:
Why only `clean` and not `full_clean`? You're skipping a bunch of validations here.

Reply:
Done